src/SDKs/Resource/Management.ResourceManager/generate.cmd

```diff
@@ -8,12 +8,12 @@ setlocal
 set autoRestVersion=0.17.0-Special-20161018
 set source=-Source https://www.myget.org/F/autorest/api/v2
 
-set resSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-resources/resources/2016-09-01/swagger/resources.json"
+set resSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-resources/resources/2017-05-10/swagger/resources.json"
```

Comment:
Please use the commit id of the last commit for your PR in the rest-api-specs repo for these values - do not refer to specs through the floating 'master' branch. For example: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/blob/2671588431bc88cb1a85899570f6b1bef17ccd11/arm-resources/locks/2016-09-01/swagger/locks.json